internal/multistorage/stats/cache/storage_statuses.go

```diff
@@ -84,31 +84,30 @@ func (s storageStatus) applyOperationResult(alive bool, weight float64, checkTim
 			WasAlive:           s.alive(),
 			Updated:            checkTime,
 		}
-	} else {
-		return storageStatus{
-			PotentialAliveness: expMovingAverage(s.PotentialAliveness, weight),
-			ActualAliveness:    expMovingAverage(s.ActualAliveness, 0),
-			WasAlive:           s.alive(),
-			Updated:            checkTime,
-		}
+	}
+	return storageStatus{
+		PotentialAliveness: expMovingAverage(s.PotentialAliveness, weight),
+		ActualAliveness:    expMovingAverage(s.ActualAliveness, 0),
+		WasAlive:           s.alive(),
+		Updated:            checkTime,
 	}
 }
 
-const emaAlpha = 0.875
+// This exponential moving average α value allows a fully dead storage to become alive after 10 successful operations.
+const emaAlpha = 0.6
```

Comment:
Let's move this const to others at the top since these 3 constants are responsible for the same logic. You can then write an example how they are connected between each other.